Packages/InputSystem/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "com.caseydecoder.unity-atoms-input-system",
+  "displayName": "Unity Atoms Input System",
+  "version": "4.3.0",
+  "unity": "2018.3",
+  "description": "Bla Bloe Bladiebla",
```

Comment:
Maybe the description should be a little bit more descriptive ;). Eg: Use Unity Atoms with Unity's Input System